### PR TITLE
Adding lessons files

### PR DESCRIPTION
**Pull request by cameliaenache**

### CODE REVIEW

File: coffee_time/coffee.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    This is a new_coffee class.
+    """
+
+    def __init__(self, coffee_spoons, sugar, water):
+        self.coffee_spoons = coffee_spoons
+        self.sugar = sugar
+        self.water = water
+
+    def prepare_coffee(self):
+        cantitate_totala_cafea = 3*self.coffee_spoons + 2*self.sugar + 5*self.water
```
Comment (flavia-c) [inline coffee_time/coffee.py]: Variables should be in English

File: cafelatte/cafelatte.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    def __init__(self, coffee_spoons, sugar, water, milk):
+        super().__init__(coffee_spoons=coffee_spoons, sugar=sugar, water=water)
+        self.milk = milk
+
+    def prepare_coffee(self):
+        cantitate_totala_cafea = super().prepare_coffee() + self.milk
+        return cantitate_totala_cafea
+
+    def prepare_time_coffee(self):
+        time_to_prepare = math.pi*math.sin(self.coffee_spoons) + math.cos(self.sugar) + self.water + 2 * self.milk
```
Comment (flavia-c) [inline cafelatte/cafelatte.py]: Method should have inherited prepare_time_coffee from Coffee class + milk

File: coffee_and_cafelatte/coffee_cafelatte.py [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+if __name__ == '__main__':
+
+    espresso = Coffee(1, 0, 2)
+    espresso_latte = Cafelatte(1, 0, 2, 2)
+
+    espresso.prepare_time_coffee()
+    print(espresso.prepare_coffee())
+    espresso_latte.prepare_time_coffee()
+    print(espresso_latte.prepare_coffee())
```
Comment (flavia-c) [inline coffee_and_cafelatte/coffee_cafelatte.py:13]: Optional step - to print the energy

File: cafelatte/cafelatte.py (hunk 2 above)
Comment (cameliaenache) [reply cafelatte/cafelatte.py]: But the requirement was to print at the end using just this method...

File: cafelatte/cafelatte.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
+from coffee_time.coffee import Coffee
+import math
```
Comment (CosminTraistaru) [inline cafelatte/cafelatte.py]: Imports should be grouped in the following order:
- standard library imports
- related third party imports
- local application/library specific imports
https://www.python.org/dev/peps/pep-0008/#imports

File: cafelatte/cafelatte.py [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+    def __init__(self, coffee_spoons, sugar, water, milk):
+        super().__init__(coffee_spoons=coffee_spoons, sugar=sugar, water=water)
+        self.milk = milk
+
+    def prepare_coffee(self):
+        cafelatte_quantity = super().prepare_coffee() + self.milk
+        return cafelatte_quantity
+
+    def prepare_time_coffee(self):
+        time_to_prepare = math.pi*math.sin(self.coffee_spoons) + math.cos(self.sugar) + self.water + 2 * self.milk
```
Comment (CosminTraistaru) [inline cafelatte/cafelatte.py]: This line is too long. It is recommended that you line has a maximum of 79 characters.
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Comment (cameliaenache) [reply cafelatte/cafelatte.py]: Some teams strongly prefer a longer line length. For code maintained exclusively or primarily by a team that can reach agreement on this issue, it is okay to increase the nominal line length from 80 to 100 characters.
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Comment (CosminTraistaru) [reply cafelatte/cafelatte.py]: Very good argument! But your line is 111 characters long.

Comment (cameliaenache) [reply cafelatte/cafelatte.py]: Why doesn't PyCharm signal this? :D

Comment (CosminTraistaru) [reply cafelatte/cafelatte.py]: It does!

File: coffee_time/coffee.py [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+        self.coffee_spoons = coffee_spoons
+        self.sugar = sugar
+        self.water = water
+
+    def prepare_coffee(self):
+        coffee_quantity = 3*self.coffee_spoons + 2*self.sugar + 5*self.water
+        return coffee_quantity
+
+    def prepare_time_coffee(self):
+        time_to_prepare = math.pi*math.sin(self.coffee_spoons) + math.cos(self.sugar) + self.water
```
Comment (CosminTraistaru) [inline coffee_time/coffee.py]: pep8 recommends to leave spaces between operators.
https://www.python.org/dev/peps/pep-0008/#other-recommendations

**Review by CosminTraistaru — APPROVED** [review]

**Pull request merged**